styles.css

```diff
@@ -0,0 +1,428 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap");
+
+@media screen and (min-width: 374px) and (max-width: 768px) {
```

Comment:
[OPTIONAL]
- [ ] Since we are using the "mobile first" approch, It would be nice to not use media query (`@media screen`) for now. You can use responsive values in your CSS rules, like percentages (i.e. width: 80%) to make the page responsive